README.md

```diff
@@ -1138,7 +1132,7 @@ price. Stock options are the most common way early-stage companies grant equity.
 🄳 The **strike price (or exercise price)** is the fixed price per share at which stock can
 be purchased, as set in the stock option agreement.
 The strike price is generally set lower (often much lower) than what people expect will be
-the *future* value of the stock, which means down the road, selling the stock could be
+the *future* value of the stock, which means selling the stock down the road could be
```

Comment:
Nit: Sounds less clear to my ear, at least (how do you sell a stock down the road? is the NYSE down the road?). How about "eventually".

Reply:
'twas in a comma clause before, I just moved it. If adding "eventually" should read: "which means eventually selling the stock could be profitable"

Reply:
Fixed.